### DISCUSSION TITLE
Is onnxruntime1.17.0+ only supports CUDA 12?

### DISCUSSION THREAD

**Discussion by duanshengliu** [microsoft/onnxruntime#19987] [Release Q&A]
Hi, my cuda environment is as follows with 8 A100 GPU:

> +-----------------------------------------------------------------------------+
> | NVIDIA-SMI 470.161.03   Driver Version: 470.161.03   CUDA Version: 11.4     |
> |-------------------------------+----------------------+----------------------+

it works well in `onnxruntime-gpu 1.16.3` &  `python 3.9.13`.  I want to upgrade `onnxruntime-gpu` to 1.17.1, so I use `pip install onnxruntime-gpu==1.17.1` but I get the following error message:
> ERROR: Could not find a version that satisfies the requirement onnxruntime-gpu==1.17.1 (from versions: 1.7.0, 1.8.0, 1.8.1, 1.9.0, 1.10.0, 1.11.0, 1.11.1, 1.12.0, 1.12.1, 1.13.1, 1.14.0, 1.14.1, 1.15.0, 1.15.1, 1.16.0, 1.16.1, 1.16.2, 1.16.3)
> ERROR: No matching distribution found for onnxruntime-gpu==1.17.1

What I'm wondering is whether `onnxruntime1.17.0+` only supports CUDA 12. By the way, I can install `onnxruntime 1.17.1` with  `pip install onnxruntime==1.17.1`

**Comment by snnn** [comment]
What's your operating system version?

**Reply by duanshengliu** [reply]
Hi，my operating system `Ubuntu 18.04.5 LTS (GNU/Linux 4.15.0-200-generic x86_64)`.

**Reply by snnn** [reply]
It is EOL. So we stopped supporting it. Please upgrade it to a newer version.

**Reply by duanshengliu** [reply]
Got it, thanks.
